mne/channels/layout.py

```diff
@@ -777,9 +780,15 @@ def read_montage(kind, ch_names=None, path=None, scale=True):
     path : str | None
         The path of the folder containing the montage file. Defaults to the
         mne/channels/data/montages folder in your mne-python installation.
-    scale : bool
-        Apply useful scaling for out the box plotting using montage.pos.
-        Defaults to True.
+    unit : 'm' | 'mm'
+        If 'mm', points will be rescaled to 'm'.
+        Default is 'm'.
```

Comment:
+1 adding to the documentation since I don't know all the units of the montages listed.